layers/meta-balena-raspberrypi/recipes-kernel/linux/linux-raspberrypi/0004-mmc-pwrseq-Repurpose-for-Marvell-SD8777.patch

```diff
@@ -0,0 +1,122 @@
+From ae3314ccf309e25b0b6617655da43ae23757d761 Mon Sep 17 00:00:00 2001
+From: Zahari Petkov <zahari@balena.io>
+Date: Thu, 15 Aug 2019 12:19:02 +0200
+Subject: [PATCH] mmc: pwrseq: Repurpose for Marvell SD8777
+
+Repurposes the mmc-pwrseq-sd8787 driver to be compatible with SD8777 instead.
+
+The original version has a reset pin, which does not exit for this chipset. The
```

Comment:
NIT: type here: `exit` instead of `exists` ?

Reply:
@floion done